src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User userWithBadAge = new User();
+    private User userWithBadPassword = new User();
+    private User goodUser = new User();
+    private User sameUser = new User();
+
+    @BeforeEach
+    public void setUp() {
+        goodUser.setAge(18);
+        goodUser.setLogin("login1245");
+        goodUser.setPassword("123456");
+        sameUser.setAge(22);
+        sameUser.setLogin("login4626");
+        sameUser.setPassword("12463578");
+        userWithBadAge.setAge(17);
+        userWithBadAge.setLogin("login12453");
+        userWithBadAge.setPassword("123456");
+        userWithBadPassword.setAge(22);
+        userWithBadPassword.setLogin("login12356");
+        userWithBadPassword.setPassword("123");
+    }
+
+    @Test
+    public void registerGoodUserOK() {
```

Comment:
Let's rename your test method according to common rules
Exemple: _methodUnderTest_state_expectedBehavior_